### ISSUE TITLE
incorrect op-spacing on generics

### ISSUE THREAD

**Issue by oliverdain** [shyiko/ktlint#34]
This expression:

```
override val formatClass: Class<out Format<*>>
        get() = super<MultiFileMixedTransform>.formatClass
```

generates:

```
RandomForestTransform.kt:25:24: Missing spacing after "<"
RandomForestTransform.kt:25:47: Missing spacing before ">"
```

which I believe is an error. I think it thinks the `<` and `>` are comparison operators rather than generics.

**Comment by shyiko** [comment]
@oliverdain Thank you! Fixed in [0.5.1](https://github.com/shyiko/ktlint/blob/master/changelog.md#051---2017-02-28) (Maven Central sync might take up to ~20 minutes).

**Comment by oliverdain** [comment]
Wow! That was FAST! Thanks.
